performance_manager/lib/l0_rt_vehicle_positions.py

```diff
@@ -406,6 +409,12 @@ def process_vehicle_positions(db_manager: DatabaseManager) -> None:
 
             new_events = transform_vp_timestamps(new_events)
             sizes["merge_events_count"] = new_events.shape[0]
+            min_ts_processed = min(
+                min_ts_processed, int(new_events["timestamp_start"].min())
+            )
+            max_ts_processed = max(
+                max_ts_processed, int(new_events["timestamp_end"].max())
+            )
```

Comment:
Same as getting timestamp from `VehiclePositions` but `TripUpdates` only has `timestamp_start` for stop data.